en/identity-server/next/docs/apis/restapis/organization-discovery-config-mgt.yaml

```diff
@@ -91,6 +91,55 @@ paths:
                 ]
             }'
       x-codegen-request-body-name: body
+    put:
+      tags:
+        - Discovery
+      summary: Update organization discovery configuration.
+      description: |
+        This API provides the capability to update discovery configuration of the primary organization. <br>
+
+        <b>Scope(Permission) required:</b> `internal_organization_config_update`
```

Comment:
This API needs to be added to Asgardeo API docs as well right?
May be we can change the sample